addons/sourcemod/scripting/vsh/bosses/boss_demorobot.sp

```diff
@@ -156,18 +156,25 @@ methodmap CDemoRobot < SaxtonHaleBase
 	{
 		if (strncmp(sample, "vo/", 3) == 0)
 		{
+			if (StrContains(sample, "vo/mvm/", false) == 0)
+				return Plugin_Continue;
+			
 			char file[PLATFORM_MAX_PATH];
 			strcopy(file, PLATFORM_MAX_PATH, sample);
 			ReplaceString(file, sizeof(file), "vo/", "vo/mvm/norm/", false);
+			ReplaceString(file, sizeof(file), "demoman_", "demoman_mvm_", false);
```

Comment:
Would be quicker if single `ReplaceString` from `vo/demoman_` to `vo/mvm/norm/demoman_mvm_`, unless if that doesn't work on voicelines i do not know of